app/views/posts/index.html.erb

```diff
@@ -1 +1,50 @@
-<h1>Here is the username and list of comments for each post!</h1>
\ No newline at end of file
+<div class="sub-container"  style="max-width: 540px;">
+    <div class="card mb-3">
+        <div class="row g-0">
+            <div class="col-md-4">
+            <img src=<%= @user.photo %> class="img-fluid img-thumbnail rounded-start" alt=<%= @user.name %>>
+        </div>
+        <div class="col-md-8">
+            <div class="card-body">
+             <h5 class="card-title"><%= @user.name %></h5>
+             <p class="card-text text-end">Number of posts: <%= @user.posts_counter %> </p>
+            </div>
+        </div>
+        </div>
+    </div>
+
+    <div class="card mb-2">
+    <div class="card-body">
+        <h6 class="card-title">Bio</h6>
+        <p class="card-text"><%= @user.bio %></p>
+    </div>
+    </div>
+
+    <% if @posts.present? %>
+    <% @posts.each do |post| %>
+    <div class="card mb-2">
+        <%= link_to user_post_path(@user, post), class: 'no-link' do %>
+        <div class="card-body">
+            <h6 class="card-title"><%= post.title %> </h6>
+            <p class="card-text"><%= post.text %> </p>
+            <p class="card-link text-end">Comment: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>  </p>
+        </div>
+        <% end %>
+    </div>
+
+      <% post.comments.each do |comment| %>
```

Comment:
- [ ] Kindly use the `most_recent_comments` method that you have created in the `Blog App - processing data in models` project in the `post#index` view instead of displaying all comments to match the wireframe and as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.).

Reply:
Thanks for your nice feedback.👌
Done ✅